### PR TITLE
Added docstrings

### PR DESCRIPTION
**Pull request by AkhilYadavPadala**
@OfficialAhmed 
Added docstrings for improving code consistency and readability.

### CODE REVIEW

File: Calculator with simple ui.py [hunk 1]
```diff
@@ -1,19 +1,59 @@
 # Program make a simple calculator
 
-# This function adds two numbers
 def add(x, y):
```
Comment (NitkarshChourasia) [inline Calculator with simple ui.py:3]: Add more functions or just remove the file.

File: Calculator with simple ui.py [hunk 2]
```diff
@@ -1,19 +1,59 @@
...
+    8
+    >>> multiply(3, 3)
+    9
+    >>> multiply(9, 9)
+    81
+    """
     return x * y
 
-# This function divides two numbers
 def divide(x, y):
```
Comment (NitkarshChourasia) [inline Calculator with simple ui.py:45]: Add error handling if divided by zero.
if num1 = n, where n > 0 and num2 = 0, then error will be returned. Handle it.

File: Calculator with simple ui.py [hunk 3]
```diff
@@ -1,19 +1,59 @@
...
+    4
+    >>> subtract(4, 9)
+    -5
+    """
     return x - y
 
-# This function multiplies two numbers
 def multiply(x, y):
+    """
+    This function multiplies two numbers.
```
Comment (NitkarshChourasia) [inline Calculator with simple ui.py:33]: use class with simple name for the class, and make this methods inside it.

File: Calculator with simple ui.py [hunk 4]
```diff
@@ -1,19 +1,59 @@
...
-# This function divides two numbers
 def divide(x, y):
+    """
+    This function divides two numbers.
+
+    Examples:
+    >>> divide(4, 4)
+    1
+    >>> divide(6, 3)
+    2
```
Comment (NitkarshChourasia) [inline Calculator with simple ui.py:53]: make it oop based, and use 
```python
if __name__ == "__main__":
        main()
```

File: Calculator with simple ui.py [hunk 5]
```diff
@@ -1,19 +1,59 @@
 # Program make a simple calculator
```
Comment (NitkarshChourasia) [inline Calculator with simple ui.py:2]: add other calculation type, make it worthwhile to contribute.

Comment (NitkarshChourasia) [inline Calculator with simple ui.py:2]: while making class, add docstring to it.

**Review by NitkarshChourasia — CHANGES_REQUESTED** [review]
Not good, a program to even submit to a repo.

**Review by OfficialAhmed — APPROVED** [review]
Can be improved but overall good work

**Review by NitkarshChourasia — APPROVED** [review]
Yes, overall a __good start__.
Keep up the good work.

**Comment by AkhilYadavPadala** [comment]
@geekcomputers Could you add hacktoberfest , hacktoberfest-accepted labels to my PR?

**Pull request merged**